java/device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/DeviceClientConfig.java

```diff
@@ -79,8 +79,7 @@ public DeviceClientConfig(String iotHubHostname, String deviceId,
         this.deviceKey = deviceKey;
         // Codes_SRS_DEVICECLIENTCONFIG_25_017: [**The constructor shall save sharedAccessToken.**] **
         this.sharedAccessToken = sharedAccessToken;
-        DefaultCertificate cert = new DefaultCertificate();
-        this.pathToCertificate = cert.getDefaultCertificate();
+
```

Comment:
If this is a bug fix, please provide in a separate PR since it is not related to the back off implementation.